network/utils.go

```diff
@@ -242,3 +242,33 @@ func isFlagSet(name string, flagset *flag.FlagSet) bool {
 	})
 	return set
 }
+
+// GetDBURLHost return DB host from MySQL/PostgreSQL connection string to use as SNI
```

Comment:
lets note that it parses MySQL/PostgreSQL driver specific connection strings. Because it can be changed in the future.